js/libs/form.js

```diff
@@ -1095,6 +1095,7 @@ Fliplet().then(function() {
             },
             field: function(key) {
               var field = $form.getField(key);
+              var instance = _.find($form.$children, { name: field.name });
```

Comment:
If you look the line after this, you'll find that we can't assume `field` is retrieved from the line above, so you can't assume `field.name` is valid.

You should also add a check to make sure the instance is found, or return an error (with the same message as below).

Also, let's define this as `$field` to imply it's a Vue object representing the field.

Then, update https://github.com/Fliplet/fliplet-widget-form-builder/blob/a5a3763524b670471682a896d213758838772a5a/js/libs/form.js#L1074 to also include `$instance: $form`.

Reply:
For consistency. Then we can update documentation so that `$instance` (with a `$`) is always a Vue instance object.